src/components/Task/MassActions/ConfirmationModal/MassActionsTasksConfirmationModal.tsx

```diff
@@ -20,23 +27,71 @@ export const MassActionsTasksConfirmationModal: React.FC<
 > = ({ open, idsCount, action, setOpen, onConfirm }) => {
   const { t } = useTranslation();
 
+  // KG: useState for checkbox
+  const [checked, setChecked] = useState(false);
+
+  const shouldConfirmDeletion = action === 'delete' && idsCount > 5;
+
+  const handleChange = (event) => {
+    setChecked(event.target.checked);
+  };
+
+  // KG: get correct task message based on action
+  function renderMessage() {
+    if (action === 'complete' || !shouldConfirmDeletion) {
```

Comment:
Can we turn the `action` property into a enum. That way it becomes easier to use in if statements.

Research enum in TS here:  https://www.typescriptlang.org/docs/handbook/enums.html